pkg/kubelet/kubelet.go

```diff
@@ -901,6 +907,74 @@ func (kl *Kubelet) setupDataDirs() error {
 	return nil
 }
 
+func (kl *Kubelet) getPodVolumes(podUID types.UID) ([]volumeTuple, error) {
+	podVolDir := kl.getPodVolumesDir(podUID)
+	volumeDirs, err := ioutil.ReadDir(podVolDir)
+	if err != nil {
+		glog.Errorf("Could not read directory %s: %v", podVolDir, err)
+	}
+	volumes := []volumeTuple{}
+	for _, volumeDir := range volumeDirs {
+		plugName := volumeDir.Name()
+		glog.Infof("get mounted volume plugName %q", plugName)
```

Comment:
You can add "WIP: " to the PR title to let reviewers know the PR is not ready yet :-)